README.md

```diff
@@ -13,7 +13,7 @@
 [![CircleCI](https://img.shields.io/circleci/build/gh/sqlfluff/sqlfluff/master?style=flat-square&logo=CircleCI)](https://circleci.com/gh/sqlfluff/sqlfluff/tree/master)
 [![ReadTheDocs](https://img.shields.io/readthedocs/sqlfluff?style=flat-square&logo=Read%20the%20Docs)](https://sqlfluff.readthedocs.io)
 
-Bored of not having a good SQL linter that works with whichever dialiect you're
+Bored of not having a good SQL linter that works with whichever dialect you're
```

Comment:
I _just_ noticed this as well the other day, thanks for fixing!

Reply:
😄  I thought this was intentionally left there for new comers!